src/hotspot/share/opto/output.cpp

```diff
@@ -748,14 +748,14 @@ void PhaseOutput::FillLocArray( int idx, MachSafePointNode* sfpt, Node *local,
   // Is it a safepoint scalar object node?
   if (local->is_SafePointScalarObject()) {
     SafePointScalarObjectNode* spobj = local->as_SafePointScalarObject();
-
-    ObjectValue* sv = sv_for_node_id(objs, spobj->_idx);
+    ObjectValue* sv = (ObjectValue*) sv_for_node_id(objs, spobj->_idx);
     if (sv == nullptr) {
       ciKlass* cik = t->is_oopptr()->exact_klass();
       assert(cik->is_instance_klass() ||
-             cik->is_array_klass(), "Not supported allocation.");
+            cik->is_array_klass(), "Not supported allocation.");
```

Comment:
Why spacing changed?

Reply:
The identation level was incorrect before.